app/src/main/java/ai/elimu/appstore/synchronization/AppListAdapter.java

```diff
@@ -58,10 +58,15 @@ public AppListAdapter(List<Application> applications,
     }
 
     @Override
-    public void onBindViewHolder(final ViewHolder holder, int position) {
+    public void onBindViewHolder(final ViewHolder holder, final int position) {
 
         final Application application = applications.get(position);
         holder.textPkgName.setText(application.getPackageName());
+        holder.pbDownload.setVisibility(application.isDownloading() ? View.VISIBLE : View.GONE);
```

Comment:
I don't understand what `pbDownload` means. Can you change the name of that variable to something more meaningful?